tests/queries/0_stateless/02314_csv_tsv_skip_first_lines.sql

```diff
@@ -0,0 +1,12 @@
+-- Tags: no-parallel
```

Comment:
Why?

Reply:
Because we work with local files in this test and parallel runs of this test can lead to different output.